src/trigger.c

```diff
@@ -0,0 +1,292 @@
+#include "common.h"
+
+void default_trigger_delegate(s32* arg0) {
+    arg0[0] |= 2;
+}
```

Comment:
Based on the function name we can prototype this `(Trigger* self)` and set flags in the body:
```suggestion
void default_trigger_delegate(Trigger* self) {
    self->flags |= 2;
}
```